_data/meltano/extractors/tap-jaffle-shop/meltanolabs.yml

```
@@ -0,0 +1,55 @@
+capabilities:
+- about
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: The Jaffle Shop Generator, aka "jafgen" is a python-based data generated
+  created by dbt Labs.
+domain_url: https://github.com/dbt-labs/jaffle-shop-generator
+executable: tap-jaffle-shop
+keywords:
+- meltano_sdk
+label: Jaffle Shop Generator
+logo_url: /assets/logos/extractors/jaffle-shop.png
+maintenance_status: active
+name: tap-jaffle-shop
+namespace: tap_jaffle_shop
+next_steps: ''
+pip_url: git+https://github.com/MeltanoLabs/tap-jaffle-shop.git
+repo: https://github.com/MeltanoLabs/tap-jaffle-shop
+settings:
+- description: "'True' to enable schema flattening and automatically expand nested\
+    \ properties."
+  kind: boolean
+  label: Flattening Enabled
+  name: flattening_enabled
+- description: The max depth to flatten schemas.
+  kind: integer
+  label: Flattening Max Depth
+  name: flattening_max_depth
+- description: User-defined config values to be used within map expressions.
+  kind: object
+  label: Stream Map Config
+  name: stream_map_config
+- description: Config object for stream maps capability. For more information check
+    out [Stream Maps](https://sdk.meltano.com/en/latest/stream_maps.html).
+  kind: object
+  label: Stream Maps
+  name: stream_maps
+- description: A name prefix to apply to all streams. Note that the dash ('-') character
+    will be interpreted by many targets as a delimiter between schema and table name.
+  kind: string
+  label: Stream Name Prefix
+  name: stream_name_prefix
```

```suggestion
  name: stream_name_prefix
  value: "tap_jaffle_shop-raw_"
```

So users will know from looking at the hub that table/stream names are prefixed by "raw_" by default.

---

@aaronsteers theres `value` and `placeholder`, did you want it to default to that value or just show an example?

---

Let's default to the value. That's the same value that the tap provides, but this overriding that is fine, and the transparency is preferable to them having to go check the tap for its built-in defaults.